From: Debian Python Modules Team
 <python-modules-team@lists.alioth.debian.org>
Date: Sun, 17 Nov 2019 15:23:56 -0500
Subject: Sphinx-no-spanning-cells

Work around "Column or row spanning cells are not implemented"

writing output... [ 88%] sqlstate_errors

Exception occurred:
  File "/usr/lib/python3/dist-packages/sphinx/writers/text.py", line 623, in visit_entry
    raise NotImplementedError('Column or row spanning cells are '
NotImplementedError: Column or row spanning cells are not implemented.
---
 doc/Makefile       |  2 +-
 doc/src/errors.rst | 14 --------------
 2 files changed, 1 insertion(+), 15 deletions(-)

diff --git a/doc/Makefile b/doc/Makefile
index eee20eb..acbc989 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -19,7 +19,7 @@ html: src/sqlstate_errors.rst
 	cp -r src/_build/html .
 
 src/sqlstate_errors.rst: ../psycopg/sqlstate_errors.h
-	$(PYTHON) src/tools/make_sqlstate_docs.py $< > $@
+	#$(PYTHON) src/tools/make_sqlstate_docs.py $< > $@
 
 doctest:
 	$(MAKE) PYTHON=$(PYTHON) -C .. package
diff --git a/doc/src/errors.rst b/doc/src/errors.rst
index f2871d9..281ddad 100644
--- a/doc/src/errors.rst
+++ b/doc/src/errors.rst
@@ -65,17 +65,3 @@ idiomatic error handler:
             cur.execute("LOCK TABLE mytable IN ACCESS EXCLUSIVE MODE NOWAIT")
         except psycopg2.errors.lookup("55P03"):
             locked = True
-
-
-SQLSTATE exception classes
---------------------------
-
-The following table contains the list of all the SQLSTATE classes exposed by
-the module.
-
-Note that, for completeness, the module also exposes all the
-:ref:`DB-API-defined exceptions <dbapi-exceptions>` and :ref:`a few
-psycopg-specific ones <extension-exceptions>` exposed by the `!extensions`
-module, which are not listed here.
-
-.. include:: sqlstate_errors.rst
